app/shop.py

```diff
@@ -0,0 +1,68 @@
+import json
+
+from app.customer import Customer
+
+
+def open_file(value: str) -> dict:
+    with open("app/config.json", "r") as file_out:
+        data = json.load(file_out)
+    data_shops = data["shops"]
+    fuel = data["FUEL_PRICE"]
+    data_customers = data["customers"]
+    if value == "customers":
+        return data_customers
+    if value == "fuel":
+        return fuel
+    if value == "shops":
+        return data_shops

```

Comment:
It's not good that you open this file every time when you want to get values, it takes extra time. Open it only once, and get all data that you needed.